e2e-test/src/test/resources/pluginDataCyAttributes.properties

```diff
@@ -0,0 +1,31 @@
+referenceName=referenceName
+projectId=project
+datasetProjectId=datasetProject
+dataset=dataset
+table=table
+truncateTable=switch-truncateTable
+truncateTableMacroInput=truncateTable
+updateTableSchema=switch-allowSchemaRelaxation
+updateTableSchemaMacroInput=allowSchemaRelaxation
+outputSchemaMacroInput=Output Schema-macro-input
+path=path
+pathSuffix=suffix
+format=select-format
+formatMacroInput=format
+sampleSize=sampleSize
+delimiter=delimiter
+skipHeader=switch-skipHeader
```

Comment:
please remove configs that are not relevant to date transform plugin

Reply:
Removed configs that are not relevant to date transform plugin